'mvn test' on parent project fails

### How to Reproduce
```mvn clean test``` on parent project

### Expected behavior
All tests pass

### Observed behavior
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.2:unpack-dependencies (unpack) on project molgenis-platform-integration-tests: Error unpacking file: ...\molgenis\molgenis-data-elasticsearch\target\test-classes to: ...\Dev\molgenis\molgenis-platform-integration-tests\target\test-classes
[ERROR] org.codehaus.plexus.archiver.ArchiverException: The source must not be a directory.
```

--- a/molgenis-api-tests/pom.xml
+++ b/molgenis-api-tests/pom.xml
@@ -169,7 +169,7 @@
         <executions>
           <execution>
             <id>unpack</id>
-            <phase>generate-resources</phase>
+            <phase>package</phase>
             <goals>
               <goal>unpack-dependencies</goal>
             </goals>

--- a/molgenis-platform-integration-tests/pom.xml
+++ b/molgenis-platform-integration-tests/pom.xml
@@ -108,7 +108,7 @@
         <executions>
           <execution>
             <id>unpack</id>
-            <phase>generate-resources</phase>
+            <phase>package</phase>
             <goals>
               <goal>unpack-dependencies</goal>
             </goals>

